core/jazz_environment-event-handler/index.js

```diff
@@ -491,29 +499,42 @@ function getServiceDetails(eventPayload, configData, authToken) {
   return new Promise((resolve, reject) => {
     var apiEndpoint = `${configData.BASE_API_URL}${configData.SERVICE_API_RESOURCE}?service=${eventPayload.service}&domain=${eventPayload.domain}&isAdmin=true`;
     var svcPayload = getSvcPayload("GET", null, apiEndpoint, authToken);
-
-    processRequest(svcPayload)
-      .then(result => { return resolve(result); })
-      .catch(err => {
-        logger.error("getServiceDetails failed: " + JSON.stringify(err));
-        var error = handleError(failureCodes.PR_ERROR_5.code, failureCodes.PR_ERROR_5.message);
-        return reject(error);
-      });
+    if (eventPayload.service === 'ui' && eventPayload.domain === 'jazz') {
+      resolve();
+    } else {
+      processRequest(svcPayload)
+        .then(result => { return resolve(result); })
+        .catch(err => {
+          logger.error("getServiceDetails failed: " + JSON.stringify(err));
+          var error = handleError(failureCodes.PR_ERROR_5.code, failureCodes.PR_ERROR_5.message);
+          return reject(error);
+        });
+    }
   });
 }
 
 var triggerBuildJob = function (result, payload, configData) {
   return new Promise((resolve, reject) => {
-    var output = JSON.parse(result);
-    if (!output.data && !output.data.services && output.data.services.length > 0) {
-      logger.error("Service details not found in service catalog: " + JSON.stringify(output));
-      var error = handleError(failureCodes.PR_ERROR_5.code, failureCodes.PR_ERROR_5.message);
-      return reject(error);
+    var output
```

Comment:
missing semi-colon here and other places between 518 - 525

Reply:
done